cmd/krel/cmd/anago/push.go

```diff
@@ -65,7 +64,7 @@ func init() {
 	)
 
 	pushCmd.PersistentFlags().StringVar(
-		&version,
+		&pushOpts.Version,
 		"version",
```

Comment:
What is the difference between `version` and `build-version` flags in this command?

Reply:
The build version is the jenkins build version. The `version` is the target version, so for example if we cut a new patch release, we will call `krel anago push` two times with the same build version, but a different `version` like `v1.19.3` and `v1.19.4-rc.0`.

We should fix this with krel stage/release…

Reply:
That's what I was thinking :) Maybe also some more context in these flag descriptions would be nice if possible